cypress/test/editor/CodeBlock.cy.js

```diff
@@ -0,0 +1,14 @@
+import { mount } from 'cypress/react'
+import CodeBlock from '../../../components/editor/CodeBlock'
+
+describe('CodeBlock component', () => {
+  it('should render correctly', () => {
+    mount(
+      <CodeBlock>
+        {`const message = 'Hello, World!';\nconsole.log(message);`}
+      </CodeBlock>
+    )
+    cy.get('.bg-code-editor-dark').should('exist')
+   
+  })
```

Comment:
Add the test for `Copy to Clipboard functionality`.
Resources:
 - https://stackoverflow.com/questions/61650737/how-to-fetch-copied-to-clipboard-content-in-cypress
 - [https://medium.com/@mohammadjeeshan/read-clipboard-text-in-cypress-3381e61f14c4](https://medium.com/@mohammadjeeshan/read-clipboard-text-in-cypress-3381e61f14c4)

Reply:
Done